spec/broadside/target_spec.rb

```diff
@@ -30,24 +28,21 @@
     it_behaves_like 'valid_configuration?', true,  env_files: ['file', 'file2']
 
     it_behaves_like 'valid_configuration?', true,  command: nil
-    it_behaves_like 'valid_configuration?', true,  command: ['bundle', 'exec', 'resque:work']
-    it_behaves_like 'valid_configuration?', false, command: 'bundle exec rails s'
+    it_behaves_like 'valid_configuration?', true,  command: %w(do something)
```

Comment:
FWIW i am super annoyed that you can't use `let(:command) { %w(do something) }` in an `it_behaves_like` expression.  oh well.